docs/root/version_history/current.rst

```diff
@@ -94,6 +94,7 @@ New Features
 * grpc-json: support specifying `response_body` field in for `google.api.HttpBody` message.
 * hds: added :ref:`cluster_endpoints_health <envoy_v3_api_field_service.health.v3.EndpointHealthResponse.cluster_endpoints_health>` to HDS responses, keeping endpoints in the same groupings as they were configured in the HDS specifier by cluster and locality instead of as a flat list.
 * hds: added :ref:`transport_socket_matches <envoy_v3_api_field_service.health.v3.ClusterHealthCheck.transport_socket_matches>` to HDS cluster health check specifier, so the existing match filter :ref:`transport_socket_match_criteria <envoy_v3_api_field_config.core.v3.HealthCheck.transport_socket_match_criteria>` in the repeated field :ref:`health_checks <envoy_v3_api_field_service.health.v3.ClusterHealthCheck.health_checks>` has context to match against. This unblocks support for health checks over HTTPS and HTTP/2.
+* health_check: added option to use `no_traffic_healthy_interval` which allows a different no traffic interval when the cluster is healthy. See :ref:`no_traffic_healthy_interval <envoy_v3_api_msg_config.core.v3.HealthCheck.no_traffic_healthy_interval>.
```

Comment:
i would make the first `no_traffic_healthy_interval` the link and remove the last sentence